mentat/interval.py

```diff
@@ -23,7 +23,7 @@ class Interval:
     end: int | float = attr.field()
 
     def contains(self, line_number: int):
-        return self.start <= line_number < self.end
```

Comment:
@PCSwingle I added this in b/c intervals with a single line were failing to be loaded into the CodeFeature's code message

E.g. `some_file.py:1,3-5` would only read lines 3-5 with the old logic

Reply:
Hmm, I see the problem. We'll probably need to remake some adjustments to the place where we automatically create intervals, but we can work this out later.

Reply:
How about for now we keep it exclusive, but in parse_intervals a few lines up we just add one to the ending interval if it's a single line? Should be on line 12 of this file.

Reply:
I think only interval.end should be exclusive; I agree that we should convert in parse_intervals